File: CS/PopupFormCustomButtonsViewInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraEditors.Popup;
using System.Drawing;

namespace PopupContainerCustomButtonControl
{
    class PopupFormCustomButtonsViewInfo : CustomBlobPopupFormViewInfo
    {
        // custom properties
        protected Rectangle fCustomButtonsRect;
        protected Size fCustomButtonsSize;
        public Rectangle CustomButtonsRect { get { return fCustomButtonsRect; } }
        public Size CustomButtonSize { get { return fCustomButtonsSize; } }
        public const int sideIndent = 5;

        // constructor
        public PopupFormCustomButtonsViewInfo(CustomBlobPopupForm ownerPopupForm) : base(ownerPopupForm) { }

        // override methods
        protected override void CalcContentRect(Rectangle bounds)
        {
            base.CalcContentRect(bounds);
            fCustomButtonsRect = SizeBarRect;
            fCustomButtonsSize = ButtonSize;
        }
    }
}

File: VB/PopupFormCustomButtonsViewInfo.vb
Imports DevExpress.XtraEditors.Popup
Imports System.Drawing

Namespace PopupContainerCustomButtonControl

    Friend Class PopupFormCustomButtonsViewInfo
        Inherits CustomBlobPopupFormViewInfo

        ' custom properties
        Protected fCustomButtonsRect As Rectangle

        Protected fCustomButtonsSize As Size

        Public ReadOnly Property CustomButtonsRect As Rectangle
            Get
                Return fCustomButtonsRect
            End Get
        End Property

        Public ReadOnly Property CustomButtonSize As Size
            Get
                Return fCustomButtonsSize
            End Get
        End Property

        Public Const sideIndent As Integer = 5

        ' constructor
        Public Sub New(ByVal ownerPopupForm As CustomBlobPopupForm)
            MyBase.New(ownerPopupForm)
        End Sub

        ' override methods
        Protected Overrides Sub CalcContentRect(ByVal bounds As Rectangle)
            MyBase.CalcContentRect(bounds)
            fCustomButtonsRect = SizeBarRect
            fCustomButtonsSize = ButtonSize
        End Sub
    End Class
End Namespace
